addons/mail/static/src/js/tours/discuss_channel_tour.js

```diff
@@ -17,13 +17,13 @@ registry.category("web_tour.tours").add("discuss_channel_tour", {
             run: "click",
         },
         {
-            trigger: ".o-discuss-ChannelSelector input",
+            trigger: ".o_command_palette_search input",
             content: markup(_t("<p>Create a channel here.</p>")),
             tooltipPosition: "bottom",
             run: `edit SomeChannel_${new Date().getTime()}`,
         },
         {
-            trigger: ".o-discuss-ChannelSelector-suggestion",
+            trigger: ".o_command:contains('Create Channel')",
```

Comment:
Again, this cannot use translated values (without translating them) as it must work in user language.